src/spec/verifiable-payid-protocol-NC.md

```diff
@@ -0,0 +1,379 @@
+
+# Verifiable PayID Protocol for Non-Custodial Services Specification
+  The Verifiable PayID protocol is designed along the same design principles as [PAYID-PROTOCOL][].
+
+## PaymentInformation Resource as JSON Web Signatures
+  The PayID Protocol [PAYID-PROTOCOL] defines a Payment Account(s) Information Resource that contains information about a particular PayID. This document further refines this definition to allow this information to be digitally signed, and then represented as a JSON Web Signature (JWS) [RFC7515] using JWS JSON Serialization.
+
+  Below, this document further defines the structure of each JWS component, for the purposes of Verifiable PayID.
+
+### JOSE Protected Header
+  For JWS, the members of the JSON object represented by the JOSE Header describe the cryptographic operations applied to the JWS Protected header and the JWS payload and optionally additional properties of the JWS.
+
+  For a complete list of members of this object, refer to [RFC7519][] and [RFC7515][]. Following is an example of a JWT object representing JOSE header parameters for JWS JSON Serialization syntax.
+
+    {
+           "name": "identityKey",
+           "alg" : "ES256K", 
+           "typ" : "JOSE+JSON",
+           "b64" : false,
+           "crit": ["b64"],
+           "jwk" :  {
+                "kty": "EC",
+                "use": "sig",
+                "crv": "secp256k1", 
+                "x"  : "0", 
+                "y"  : "0",
+                    },
+    }
+
+#### name
+The `name` Header Parameter identifies the type of signature. It is a new OPTIONAL header parameter that is not defined in the IANA JSON Web Signature and Encryption Header Parameters Registry.
+
+#### alg
+The "alg" (algorithm) Header Parameter identifies the cryptographic algorithm used to secure the JWS. This is a required field as described in [RFC7515][]. We RECOMMEND using "ES256K" which is Elliptic Curve Digital Signature Algorithm (ECDSA) using secp256k1 curve-type and SHA-256 hash-type as defined in IANA JSON Web Signature and Encryption Header Parameters Registry.
+
+#### typ
+The "typ" (type) Header Parameter is used by JWS applications to declare the media type of the complete JWS. This is an optional field as described in [RFC7515][].
+
+#### b64
+The "b64" (base64url-encode) Header Parameter is an extension to JWS specification that determines how a payload is represented in the JWS and the JWS signing input. When the "b64" value is `false`, the payload is represented simply as the JWS Payload value with no encoding; otherwise, it is represented as ASCII(BASE64URL(JWS Payload)). This is an optional field as described in [RFC7797][].
```

Comment:
```suggestion
The `b64` (base64url-encode) Header Parameter is an extension to JWS specification that determines how a payload is represented in the JWS and the JWS signing input. When the "b64" value is `false`, the payload is represented simply as the JWS Payload value with no encoding; otherwise, it is represented as ASCII(BASE64URL(JWS Payload)). This is an optional field as described in [RFC7797][].
```